omnigibson/prims/prim_base.py

```diff
@@ -135,17 +144,21 @@ def initialized(self):
         return self._initialized
 
     @property
-    def state_size(self):
-        # This is the cached value
-        return self._state_size
+    def scene(self):
+        """
+        Returns:
+            Scene: Scene object that this prim is loaded into
```

Comment:
Can this be None?

Reply:
For the viewer camera, yes. I'll update this docstring to include None